### ISSUE TITLE
数据相关

### ISSUE THREAD

**Issue by fengyanWang** [GuangtaoLyu/FETNet#7]
非常感谢您开源这么棒的项目，通过看工程的readme，训练似乎只需要准备原始图像，去除手写的图像和对应的mask图，应该不需要准备其它数据了吧？（下图分别为原始图，mask图和涂抹图，请问mask图生成是否正确？）

![file04_hw_005](https://github.com/user-attachments/assets/198b87c9-bf27-454f-aab0-4613f3736938)
![file04_hw_005](https://github.com/user-attachments/assets/8b31012b-1ccc-487d-a454-7ea1be7ec560)
![file04_hw_005](https://github.com/user-attachments/assets/fcd9784b-307e-4d08-b6e6-8daf75fc0bbc)

**Comment by GuangtaoLyu** [comment]
是的。嫌麻烦的话，只要原始图和擦除后的就行。mask可以通过gt-img，然后调一调阈得到。 能有真实的mask应该更好一些。
